### ISSUE TITLE
Workspace should load layers

### ISSUE THREAD

**Issue by Zodiase** [openskope/skope-interface#90]
Example url:

```
http://141.142.211.232/geoserver/SKOPE/wms
    ?service=WMS
    &version=1.1.0
    &request=GetMap
    &layers=SKOPE:paleocar_1_GDD_00070101
    &styles=
    &bbox=-114.995833333,30.995833333999997,-101.995833333,42.995833334
    &width=768
    &height=708
    &srs=EPSG:4269
    &format=application/openlayers
```  

Temporal variable is encoded in the layer name.

**Comment by Zodiase** [comment]
Blocked by Nebular down-time.

**Comment by tmcphillips** [comment]
New issue to create a docker image for running GeoServer in development mode, configured  to serve small data data sets included in the same image by default?

**Comment by Zodiase** [comment]
@tmcphillips Perhaps. But I could also work on other items while waiting.

**Comment by jterstriep** [comment]
Kind of pointless. We'll have the staging server back up soon. The
geoserver isn't in development; WMS isn't in development.

**Comment by ludaesch** [comment]
According to the plan for the first test release 0.1, GeoServer is a subsystem that will be included in the first test release. So even if the staging server is back up soon (https://wiki.ncsa.illinois.edu/display/NS/NCSA+Status+Home lists 5pm today?), having a GeoServer docker image with some test data seems like a useful deliverable. After all, for engineering and other reasons, we're going to release software (not just a service behind a URL).

**Comment by jterstriep** [comment]
It's not at that level really. We're not releasing geoserver. We're pulling a geoserver image from dockerhub -- just like everyone else. I don't think we should be getting into the situation of attempting to create, document, test and support another public geoserver image on dockerhub. Other people are doing it better than we ever will.

Now the question of a local development image with sample data might make sense in some situations. In our case, I don't see it speeding the development effort so I wouldn't place it on the critical path.

**Comment by tmcphillips** [comment]
Yes, I actually meant a Compose file, not a new Dockerfile that produces a custom image for geoserver from scratch (although there might be reason to add a layer to a standard geoserver image to streamline things a bit just for practical reasons).  

One nice thing about the elasticsearch set up we have now is that we can run tests that are destructive (delete an index, add a mapping, index test data).  And one can work on both sides of the ES protocol at the same time--change the mapping on the ES side, and update the searchkit widgets on the UI side, and see how things work.  We can't easily do either with a shared, long-running ES instance in a staging area.  I was thinking of something analogous for geoserver with the expectation of similar benefits.

**Comment by Zodiase** [comment]
#93 resolves this issue.
